hw13/trigrams.py

```diff
@@ -0,0 +1,36 @@
+import random
+
+
+d = {}
+
+
+def read_text(lines):
+    for x in range(0, len(lines) - 1):
+        words = lines[x].lower().split()
+        for y in range(0, len(words) - 2):
+            d[(words[y], words[y + 1])] = [words[y + 2]]
+    generate_new_text(d)
+
+
+def generate_new_text(d):
+    new_text_story = ""
+    for x in range(1):
+        new_sentence = list(random.choice(list(d.keys())))
+        new_sentence[0] = new_sentence[0].capitalize()
+        for y in range(random.randint(5, 10)):
+            words = random.choice(list(d.keys()))
+            new_sentence.extend(words)
```

Comment:
What you are wanting to do is select two words, and then append a third word from the dictionary of trigrams based on the two word key. Next you take the previous two words and repeat the process. This is just finding random word pairs and stringing them together. Your trigram story will read much more like real sentences when functioning properly.